packages/node/src/indexer/unfinalizedBlocks.service.ts

```diff
@@ -0,0 +1,52 @@
+// Copyright 2020-2022 OnFinality Limited authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import { Injectable } from '@nestjs/common';
+import {
+  ApiService,
+  BaseUnfinalizedBlocksService,
+  Header,
+  NodeConfig,
+  StoreCacheService,
+  getLogger,
+} from '@subql/node-core';
+import { BlockWrapper, SorobanBlock } from '@subql/types-soroban';
+
+const logger = getLogger('unfinalized');
+
+export function blockToHeader(blockHeight: number): Header {
+  return {
+    blockHeight: blockHeight,
+    blockHash: blockHeight.toString(),
+    parentHash: (blockHeight - 1).toString(),
```

Comment:
Does soroban not have block hashes? Or an api to get by block hash?

Reply:
there is no api to get by block hash